glean-core/src/lib.rs

```diff
@@ -120,26 +124,25 @@ impl Glean {
         )
     }
 
-    /// Send a ping by name.
+    /// Send a ping.
     ///
     /// The ping content is assembled as soon as possible, but upload is not
     /// guaranteed to happen immediately, as that depends on the upload
     /// policies.
     ///
-    /// TODO: (Verify this is correct):
     /// If the ping currently contains no content, it will not be sent.
```

Comment:
Since we're here, can we fix the language here? I think we need to use 'assembled' in most places in which we used 'sent'